### ISSUE TITLE
Jabbing Master triggers from additional damage source

### ISSUE THREAD

**Issue by Mecar1997** [Holic75/KingmakerRebalance#74]
Example: If you have Feral Combat Training (Slam) and Jabbing Style / Jabbing Master, and shift into Elemental Body (Fire or Frost), the bonus 1d4 Energy will also trigger the bonus Jabbing Style damage.

For example:
What happens with 5 slams (Elemental Body II + Flurry of Blows II + Haste) =>
1. Melee hit for normal damage + 1d4 Burn (+2d6 Jabbing Style damage)
2. Melee hit for normal damage + 2d6 Jabbing Style damage + 1d4 Burn (+ 4d6 Jabbing Master damage)
3. Melee hit for normal damage + 4d6 Jabbing Master damage + 1d4 Burn (+ 4d6 Jabbing Master damage)
4. Melee hit for normal damage + 4d6 Jabbing Master damage + 1d4 Burn (+ 4d6 Jabbing Master damage)
5. 3Melee hit for normal damage + 4d6 Jabbing Master damage + 1d4 Burn (+ 4d6 Jabbing Master damage)

Basically, every attack is triggered the damage from Jabbing style twice. 

This interaction can also happen with any additional damage source, such as Geniekind.

**Comment by Holic75** [comment]
fixed
